gen/pb-python/pyproject.toml

```diff
@@ -0,0 +1,41 @@
+[build-system]
+requires = ["flit_core >=3.2,<4"]
+build-backend = "flit_core.buildapi"
+
+[project]
+name = "sigstore-protobuf-specs"
+version = "0.0.1"
+description = "A library for serializing and deserializing Sigstore messages"
+readme = "../../README.md"
+license = { file = "../../LICENSE" }
+authors = [
+  { name = "Sigstore Authors", email = "sigstore-dev@googlegroups.com" }
+]
+classifiers = [
+  "License :: OSI Approved :: Apache Software License",
+  "Programming Language :: Python :: 3 :: Only",
+  "Programming Language :: Python :: 3",
+  "Programming Language :: Python :: 3.7",
+  "Programming Language :: Python :: 3.8",
+  "Programming Language :: Python :: 3.9",
+  "Programming Language :: Python :: 3.10",
+  "Programming Language :: Python :: 3.11",
+  "Development Status :: 4 - Beta",
+  "Intended Audience :: Developers",
+  "Topic :: Security",
+  "Topic :: Security :: Cryptography",
+]
+dependencies = [
+  "betterproto==2.0.0b5",
+]
+requires-python = ">=3.7"
+
+[project.urls]
+Homepage = "https://pypi.org/project/sigstore-protobuf-specs/"
+Issues = "https://github.com/sigstore/protobuf-specs/issues"
+Source = "https://github.com/sigstore/protobuf-specs"
+
+[project.optional-dependencies]
+dev = [
+  "betterproto[compiler]==2.0.0b5",
```

Comment:
The developers of `betterproto` have been working on a v2.0.0 release for a few years now. We pretty much have to use a beta version as the current stable version seems to have bugs with relative imports and lacks support for `optional`.

Reply:
I'm okay with a beta version here, as long as we keep it pinned (which we are).